drivers/flash/Kconfig.siwx91x

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2024 Silicon Laboratories Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_FLASH_SILABS_SIWX91X
+	bool "Silicon Labs SiWx91x flash driver"
+	default y
+	depends on DT_HAS_SILABS_SIWX91X_FLASH_CONTROLLER_ENABLED
+	select FLASH_HAS_DRIVER_ENABLED
+	select FLASH_HAS_EXPLICIT_ERASE
+	select FLASH_HAS_PAGE_LAYOUT
+	# Flash controller is handled by the network coprocessor
+	select WISECONNECT_NETWORK_STACK
+	help
+	  Enable flash controller for flash embedded on Silicon Labs SiWx91x
+	  chips.
+
+if SOC_FLASH_SILABS_SIWX91X
+
+choice SIWX91X_FLASH_MODE
+	bool "Flash Mode"
+	default SIWX91X_FLASH_MODE_COMMON
+
+config SIWX91X_FLASH_MODE_COMMON
+	bool "Common Flash"
+
+endchoice
```

Comment:
any reason there is a symbol for just one thing?

Reply:
Not yet. I will drop it.

Reply:
I have dropped these options.